src/ioctls/system.rs

```diff
@@ -96,6 +140,14 @@ impl Kvm {
     ///
     /// See the documentation for `KVM_GET_VCPU_MMAP_SIZE`.
     ///
+    /// # Example
+    ///
+    /// ```
+    /// # use kvm_ioctls::Kvm;
+    /// let kvm = Kvm::new().unwrap();
+    /// assert!(kvm.get_vcpu_mmap_size().unwrap() > 0);
+    /// ```
+    ///
     pub fn get_vcpu_mmap_size(&self) -> Result<usize> {
```

Comment:
Since you got rid of the `get_` prefix from `get_run_size`, shouldn't the getter naming convention be consistent?

Reply:
get_run_size is named that way because of the name of the ioctl: KVM_GET_VCPU_MMAP_SIZE. All ioctl methods are named by removing the KVM_ prefix and converting it to lower case.